package.json

```diff
@@ -842,11 +847,14 @@
         "mocha-multi-reporters": "^1.1.7",
         "tslint": "^5.7.0",
         "tslint-microsoft-contrib": "5.2.1",
+        "ts-loader": "^5.3.3",
```

Comment:
with dev 0.1.4 you can remove ts-loader

Reply:
Guess not - you listed it as a devDependency instead of dependency https://github.com/Microsoft/vscode-azuretools/pull/391/files#diff-cdde3b7ab205072caac2c61bf7b935e8R49

Reply:
Whoops. https://github.com/Microsoft/vscode-azuretools/pull/393